meta-genivi-dev/recipes-extended/chromium/chromium-wayland_%.bbappend

```diff
@@ -5,6 +5,9 @@ SRC_URI += "\
     file://chromium.service \
 "
 
+# FIXME: workaround for https://at.projects.genivi.org/jira/browse/LM-2
+CHROMIUM_EXTRA_ARGS_append = " --window-size=1728,1080"
+
```

Comment:
Oh, I realize I've misunderstood your comment. The workaround should work on a 720p screen because that resolution has the same aspect ratio, and the HMI would downscale the browser surface. I reckon it's a waste, though. It wouldn't work with other resolutions, but those are generally not well supported by embedded hardware so I guess they aren't very interesting.